zombienet/0004-webrtc/test-webrtc.toml

```
@@ -0,0 +1,33 @@
+[settings]
+enable_tracing = false
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+default_command = "substrate"
+
+chain = "gen-db"
+chain_spec_path = "zombienet/0004-webrtc/chain-spec.json"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = false
```

```suggestion
  validator = false
  p2p_cert_hash = "uEiAvF2IuPZJuKkdb5E-gSM_AmumLi3P5ZR_fvZGPE3vJEA"
```

Hash of the cert used in this test.